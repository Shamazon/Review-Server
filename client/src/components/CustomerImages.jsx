import style from '../../dist/style/style.css'

var CustomerImages = (props) => {
    return (
        <div>
            <h3>Customer images</h3>
            <div className={style.customerImages}>
                <img src='https://source.unsplash.com/random/71x71' />
                <img src='https://source.unsplash.com/random/71x71' />
                <img src='https://source.unsplash.com/random/71x71' />
                <img src='https://source.unsplash.com/random/71x71' />
            </div> 
            <a className={style.underline}>See all customer images</a>
        </div>
    )
}

export default CustomerImages;