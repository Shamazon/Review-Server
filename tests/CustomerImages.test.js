import React from 'react';
import CustomerImages from '../client/src/components/CustomerImages.jsx';
import enzyme from 'enzyme';

describe('CustomerImages', () => {
  it('should be defined', () => {
    expect(CustomerImages).toBeDefined();
  });
  it('should render correctly', () => {
    const props = {
      reviews: ['test'],
    };
    const tree = enzyme.shallow(<CustomerImages {...props} />);
    expect(tree).toMatchSnapshot();
  });
});
