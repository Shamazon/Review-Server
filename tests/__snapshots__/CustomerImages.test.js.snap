// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomerImages should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <CustomerImages
    reviews={
      Array [
        "test",
      ]
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Customer images
        </h3>,
        <div
          className="customerImages"
        >
          <img
            src="https://source.unsplash.com/random/71x71"
          />
          <img
            src="https://source.unsplash.com/random/71x71"
          />
          <img
            src="https://source.unsplash.com/random/71x71"
          />
          <img
            src="https://source.unsplash.com/random/71x71"
          />
        </div>,
        <a
          className="underline"
        >
          See all customer images
        </a>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Customer images",
        },
        "ref": null,
        "rendered": "Customer images",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              src="https://source.unsplash.com/random/71x71"
            />,
            <img
              src="https://source.unsplash.com/random/71x71"
            />,
            <img
              src="https://source.unsplash.com/random/71x71"
            />,
            <img
              src="https://source.unsplash.com/random/71x71"
            />,
          ],
          "className": "customerImages",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "src": "https://source.unsplash.com/random/71x71",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "src": "https://source.unsplash.com/random/71x71",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "src": "https://source.unsplash.com/random/71x71",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "src": "https://source.unsplash.com/random/71x71",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "See all customer images",
          "className": "underline",
        },
        "ref": null,
        "rendered": "See all customer images",
        "type": "a",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Customer images
          </h3>,
          <div
            className="customerImages"
          >
            <img
              src="https://source.unsplash.com/random/71x71"
            />
            <img
              src="https://source.unsplash.com/random/71x71"
            />
            <img
              src="https://source.unsplash.com/random/71x71"
            />
            <img
              src="https://source.unsplash.com/random/71x71"
            />
          </div>,
          <a
            className="underline"
          >
            See all customer images
          </a>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Customer images",
          },
          "ref": null,
          "rendered": "Customer images",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <img
                src="https://source.unsplash.com/random/71x71"
              />,
              <img
                src="https://source.unsplash.com/random/71x71"
              />,
              <img
                src="https://source.unsplash.com/random/71x71"
              />,
              <img
                src="https://source.unsplash.com/random/71x71"
              />,
            ],
            "className": "customerImages",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "src": "https://source.unsplash.com/random/71x71",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "src": "https://source.unsplash.com/random/71x71",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "src": "https://source.unsplash.com/random/71x71",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "src": "https://source.unsplash.com/random/71x71",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "See all customer images",
            "className": "underline",
          },
          "ref": null,
          "rendered": "See all customer images",
          "type": "a",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
